src/lib/y2storage/proposal/lvm_creator.rb

```diff
@@ -102,8 +102,11 @@ def create_volume_group(planned_vg, devicegraph)
       # @param devicegraph [Devicegraph] to fetch the partitions
       def assign_physical_volumes(volume_group, part_names, devicegraph)
         partitions = part_names.map { |n| BlkDevice.find_by_name(devicegraph, n) }
+        used_devices = volume_group.lvm_pvs.map(&:plain_blk_device)
         partitions.each do |partition|
           device = partition.encryption || partition
+          next if used_devices.include?(device) # already a PV for this VG
+          device.remove_descendants
```

Comment:
You fixed it twice... so now it's wrong again:-) `used_device` is now never an `Encryption` but `device` can be.

Either you use `map(&:blk_device)` again or you move the `next if` again to it's original location.